src/main/scala/org/renci/cam/QueryService.scala

```diff
@@ -381,7 +381,9 @@ object QueryService extends LazyLogging {
     SPARQLQueryExecutor.runSelectQueryAs[SlotStuff](queryText.toQuery)
   }
 
-  private def getTRAPIQEdgePredicates(edge: TRAPIQueryEdge): RIO[ZConfig[AppConfig] with HttpClient, List[IRI]] =
+  final private case class Predicate(predicate: IRI)
```

Comment:
Sorry for continuing to be unclear - I really meant the `Predicate` class to be entirely private to the getTRAPIQEdgePredicates method. It's just a convenient way to describe the expected SELECT output, and catches all errors related to casting results to an IRI.